### PR TITLE
PODS-8821 - Add Event 24

### PR DESCRIPTION
**Pull request by LauraToddCodes**

### CODE REVIEW

File: app/models/enumeration/EventType.scala [hunk 1]
```diff
@@ -108,6 +108,10 @@ object EventType extends Enumerable.Implicits {
     override val order = 19
   }
 
+  case object Event24 extends WithName("24") with EventType {
+    override val order = 19
```
Comment (PavelVjalicin) [inline app/models/enumeration/EventType.scala:112]: Order should be 20 here.

**Review by PavelVjalicin — CHANGES_REQUESTED** [review]
Does this code change required for this ticket? What impact does it have on frontend?

**Comment by LauraToddCodes** [comment]
> Is this code change required for the ticket? What impact does it have on frontend?

It does seem to be needed, if you run the FE locally without this change then you'll get a 500 error after the Member's details page where it cannot find Event 24

**Comment by PavelVjalicin** [comment]
> > Is this code change required for the ticket? What impact does it have on frontend?
> 
> It does seem to be needed, if you run the FE locally without this change then you'll get a 500 error after the Member's details page where it cannot find Event 24

Sounds good.

**Review by PavelVjalicin — APPROVED** [review]
LGTM

**Pull request merged**